x-pack/plugins/code/public/components/symbol_tree/props.ts

```diff
@@ -4,251 +4,90 @@
  * you may not use this file except in compliance with the Elastic License.
```

Comment:
is this a file for test?

could we add `test` to the file name or put it to a directory that has name `test`?